tools/ci_connector_ops/ci_connector_ops/qa_engine/inputs.py

```diff
@@ -50,6 +49,3 @@ def fetch_adoption_metrics_per_connector_version() -> pd.DataFrame:
         "total_syncs_count",
         "sync_success_rate",
     ]]
-
-CLOUD_CATALOG = fetch_remote_catalog(CLOUD_CATALOG_URL)
```

Comment:
I removed these constants from the module because they cause unnecessary network call on module import. I preferred to use dependency injections and test fixtures to expose these (direct calls to the `fetch_remote_catalog` function.